src/locales/es.po

```diff
@@ -612,10 +626,17 @@ msgid "Count conversions within X days of an impression"
 msgstr ""
 
 #: src/user/analytics/analyticsOverview/components/MetricSelect.tsx:46
-msgid "Counted if the user clicks an ad and spends at least 10 seconds on the advertiser's website, with the website open in an active browser tab. The 10 seconds must be spent on the site after arriving by clicking the ad link, and the tab must remain open and active the entire time for the visit to count."
+#: src/user/analytics/search/metrics.ts:62
+#~ msgid "Counted if the user clicks an ad and spends at least 10 seconds on the advertiser's website, with the website open in an active browser tab. The 10 seconds must be spent on the site after arriving by clicking the ad link, and the tab must remain open and active the entire time for the visit to count."
+#~ msgstr ""
+
+#: src/user/analytics/analyticsOverview/components/MetricSelect.tsx:46
+#: src/user/analytics/search/metrics.ts:62
+msgid "Counted if the user clicks an ad and spends at least 5 seconds on the advertiser's website, with the website open in an active browser tab. The 10 seconds must be spent on the site after arriving by clicking the ad link, and the tab must remain open and active the entire time for the visit to count."
```

Comment:
i don't think this is live

Reply:
yup i was over-keen with that change, backing it out